src/main.ts

```diff
@@ -27,13 +27,16 @@ const run = (): void => {
   }
 
   try {
-    const urlToWait: string = core.getInput('deployment-url');
+    const urlToWait: string = core.getInput('deployment-url', { required: true });
```

Comment:
What does this do? Throw an error if the input is missing? I thought it was already handled by `action.yml` 🤔

Reply:
Yeah, it is already defined in `action.yml`, but it does not hurt to make sure it is defined.

Reply:
Okay, I'm still unsure of how it **behaves**, though. I expected the action not to run if something required is not provided, but that might be a mistake on my side. (might only be used for generating documentation)

I guess there is no harm in indicating it's required there then.